src/store/users/api.ts

```diff
@@ -0,0 +1,11 @@
+import { apiService } from 'services';
+
+import { IResponseApi } from '../../types';
+
+import { IUser } from './types';
+
+type IGetUsersApi = () => Promise<IResponseApi<IUser[]>>;
```

Comment:
You can move Promise<IResponseApi<>> with generic to the separate type

And rename it please. The last word should explain the object